.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+	"name": "CARMA build image",
+	"image" : "ghcr.io/usdot-fhwa-stol/carma-builds-x64:focal",
+//	"image" : "carma_cmake_builder_x64",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Uncomment the next line to run commands after the container is created.
+	"postCreateCommand": "./install_dependencies.sh",
+	// Configure tool-specific properties.
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-vscode.cpptools-extension-pack",
+				"EditorConfig.EditorConfig",
+				"twxs.cmake",
+				"ms-vscode.cmake-tools"
+			]
+		}
+	}
+
+	// Uncomment to connect as an existing user other than the container default. More info: https://aka.ms/dev-containers-non-root.
+	// "remoteUser": "devcontainer"
+}
```

Comment:
Let's remove the boilerplate comments to avoid cluttering the file.

Reply:
Good call.